### PR TITLE
Set07| Add Node Documentation for...

### PR DESCRIPTION
**Pull request by johnpierson**
### Purpose
ModelText.Depth
ModelText.Position
ModelText.Text
Viewport.BoxCenter
Viewport.BoxOutline
Viewport.LabelLineLength
Viewport.LabelOffset
Viewport.LabelOutline
Viewport.SetLabelLineLength
Viewport.Sheet
Viewport.View

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers
@achintyabhat 

### FYIs
@Amoursol

### CODE REVIEW

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.ModelText.Depth.md
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.ModelText.Depth.md:1]: Retrieved Depth is in what units?

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Viewport.LabelLineLength.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Viewport.LabelLineLength` gets the length of the viewport line, in paper space units.
+
+In the example below, all of the viewports from the selected sheet are collected and the line lengths are reported.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Viewport.LabelLineLength.md:4]: For clarity, can be rephrased as "In the example below, all of the viewports from the selected sheet are collected and the lengths of the viewport label lines are retrieved."

File: doc/distrib/NodeHelpFiles/_review/Revit.Elements.Viewport.LabelOutline_img.jpg
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/_review/Revit.Elements.Viewport.LabelOutline_img.jpg:1]: The 3D Watch node is not very clear. 
Revit Elements Viewport LabelOutline

**Review by achintyabhat — CHANGES_REQUESTED** [review]
This set is almost good to go, just two minor comments.

**Comment by johnpierson** [comment]
Sweet. All changes are made. And node samples moved from review.

File: doc/distrib/NodeHelpFiles/Revit.Elements.Viewport.LabelLineLength.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Viewport.LabelLineLength` gets the length of the viewport line, in paper space units.
+
+n the example below, all of the viewports from the selected sheet are collected and the lengths of the viewport label lines are retrieved.
```
Comment (achintyabhat) [inline doc/distrib/NodeHelpFiles/Revit.Elements.Viewport.LabelLineLength.md:4]: I is missing in the "In".

**Review by achintyabhat — APPROVED** [review]

**Pull request merged**